composer.json

```diff
@@ -21,10 +21,20 @@
     ],
     "require": {
         "php": "^7.1",
```

Comment:
Can we use PHP 7.1 here? This is a version break, if we require this in the CoreBundle stable version

Reply:
I think it should be ok now that Ubuntu supports 7.1 . We've been bashed in the past for doing this, but it was already for 7.1, and we are not requiring 7.2.

Reply:
On master, we can, but on 1.x it should be same as core 3.x